sdk/servicebus/Azure.ResourceManager.ServiceBus/src/autorest.md

```diff
@@ -201,23 +125,68 @@ directive:
     - from: swagger-document
       where: $.paths['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/privateEndpointConnections/{privateEndpointConnectionName}'].put.responses.201
       transform: $['description'] = 'Request to update Status of PrivateEndpoint Connection accepted.'
-    - rename-model:
-        from: RegenerateAccessKeyParameters
-        to: RegenerateAccessKeyContent
-    - rename-model:
-        from: ServiceBusNamespaceUpdateParameters
-        to: ServiceBusNamespaceUpdateContent
-    - from: swagger-document
-      where: $.definitions.NetworkRuleSet.properties.properties.properties.ipRules
-      transform: $['x-ms-client-name'] = 'iPRules'
-    - from: swagger-document
-      where: $.definitions.NetworkRuleSetIPRules.properties.ipMask
-      transform: $['x-ms-client-name'] = 'iPMask'
-    - from: swagger-document
-      where: $.definitions.DisasterRecovery.properties.properties.properties.provisioningState
+    - from: DisasterRecoveryConfig.json
+      where: $.definitions
+      transform: >
+        $.ArmDisasterRecovery['x-ms-client-name'] = 'ServiceBusDisasterRecovery';
+        $.ArmDisasterRecovery.properties.properties.properties.provisioningState['x-ms-enum'].name = 'ServiceBusDisasterRecoveryProvisioningState';
+        $.ArmDisasterRecovery.properties.properties.properties.role['x-ms-enum'].name = 'ServiceBusDisasterRecoveryRole';
+    - from: migrationconfigs.json
+      where: $.definitions
+      transform: >
+        $.MigrationConfigProperties['x-ms-client-name'] = 'MigrationConfiguration';
+        $.MigrationConfigProperties.properties.properties.properties.targetNamespace['x-ms-format'] = 'arm-id';
```

Comment:
`namespace` suffix but it's type is `ResourceIdentifier` is weird, consider to change the name